package.json

```diff
@@ -1,14 +1,34 @@
 {
   "type": "module",
+  "version": "1.0.0",
+  "name": "@warp-ds/migrate",
```

Comment:
Perhaps we should call it preset-migrate, to avoid any confusions? 😄

Reply:
Good point! Fixed in [a810bb3](https://github.com/warp-ds/preset-migrate/pull/3/commits/a810bb3df8e973b51dbc33e6ec8b7db0de976230)

Reply:
🙌 😄